src/store/getters.ts

```diff
@@ -528,10 +535,13 @@ export const getEntityTypesBC = (state: StateIF): EntityI[] => {
       return generateEntities(mapping[getRequestActionCd(state)])
     }
 
-    return EntityTypesBcData
+    // safe fallback
+    // continuation in uses a different list than the others
+    if (isContinuationIn(state) && isLocationBC(state)) return EntityTypesContInData
+    else return EntityTypesBcData
   } catch (err) {
     console.error('entityTypesBC() =', err) // eslint-disable-line no-console
-    return EntityTypesBcData
+    return []
```

Comment:
The old code masked any errors here. The new code should be safe and make any errors obvious.

Reply:
Note that any error here should be a code error that we should catch during testing, so displaying "nothing" is good option.

If this error was a runtime error then it should be handled differently as a regular user might see it.

Reply:
👍👍 👍 👍